addon/helpers/can-access-route.ts

```diff
@@ -1,53 +1,60 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import { EVENTS } from '@bagaar/ember-permissions/-private/config';
+import PermissionsService from '@bagaar/ember-permissions/services/permissions';
 import Helper from '@ember/component/helper';
 import { action } from '@ember/object';
 import { addListener, removeListener } from '@ember/object/events';
 import { inject as service } from '@ember/service';
 
 export default class CanAccessRouteHelper extends Helper {
-  @service('permissions') permissionsService;
+  @service('permissions') declare permissionsService: PermissionsService;
 
-  constructor() {
-    super(...arguments);
+  constructor(owner: unknown) {
+    // @ts-ignore
+    super(owner);
 
     addListener(
       this.permissionsService,
+      // @ts-ignore
       EVENTS.PERMISSIONS_CHANGED,
```

Comment:
Also a DT issue, [the type expects to be a key of the context](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/807712c981a319bd00658c56e13709e64b634372/types/ember__object/events.d.ts#L8), which doesn't make sense to me but maybe I'm misunderstanding it.